frontend/src/pages/Assignments/CreateAssignment/index.tsx

```diff
@@ -9,12 +9,15 @@ import StarterCodeDetails from "@/components/MultiStepForm/CreateAssignment/Star
 import { createAssignment, assignmentNameExists } from "@/api/assignments";
 
 import "./styles.css";
+import { useClassroomUser } from "@/hooks/useClassroomUser";
+import { ClassroomRole } from "@/types/enums";
 
 const CreateAssignment: React.FC = () => {
   const navigate = useNavigate();
 
   // Determine active classroom and organization
   const { selectedClassroom } = useContext(SelectedClassroomContext);
+  useClassroomUser(selectedClassroom?.id, ClassroomRole.PROFESSOR);
```

Comment:
Shouldn't we error here if we can't determine the classroom we're in?

Reply:
fixed